app/models/ability.rb

```diff
@@ -11,5 +12,34 @@ def initialize(user)
 
     can %i[read create update destroy], Project, user_id: user.id
     can %i[read create update destroy], Component, project: { user_id: user.id }
+
+    can %i[create], School # The user agrees to become a school-owner by creating a school.
```

Comment:
we'll need to make the permissions more lenient here to allow `read create destroy` on `SchoolClass`, for non verified purposes